### PR TITLE
(Blog) Update for Week 15

### PR DESCRIPTION
**Pull request by Disha-maker**
Please Review and Merge

### CODE REVIEW

File: _lifeofinternatpuppet/post_15.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+What a busy time of the year as we wrap up for Christmas?
+
+Important Highlights from this week:-
+- I continued working on running Litmus Acceptance Tests locally for the IAC modules before shifting them on the new Google Cloud CI.
+The modules tested this week are given below:-
+    -[puppetlabs-vcsrepo](https://github.com/puppetlabs/puppetlabs-vcsrepo)
+    -[puppetlabs-exec](https://github.com/puppetlabs/puppetlabs-exec)
+    -[puppetlabs-facter_task](https://github.com/puppetlabs/puppetlabs-facter_task)
+
+- Thank you to [Sheena](https://github.com/sheenaajay)for doing such an enjoyable session on module release prep and module release.
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_15.md:23]: nit:
```suggestion
- Thank you to [Sheena](https://github.com/sheenaajay) for doing such an enjoyable session on module release prep and module release.
```

**Review by sanfrancrisko — COMMENTED** [review]
Just one minor issue, and in addition, the PR title should be `(BLOG)` or something along those lines. `(MAINT)` is an abbreviation for maintenance, which isn't the correct category for this PR.

**Comment by Disha-maker** [comment]
@sanfrancrisko  If I check the preview of the file changed, then the changes are not applied. On the other hand, if I open the PR created last night, then new changes can be seen there but when that file is on the preview mode the new change (space after the 'for' word) is not seen. If any confusion, do reach out. Thank you :)

**Review by sanfrancrisko — APPROVED** [review]

File: _lifeofinternatpuppet/post_15.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Hello everyone,
+
+What a busy time of the year as we wrap up for Christmas?
+
+Important Highlights from this week:-
+- I continued working on running Litmus Acceptance Tests locally for the IAC modules before shifting them on the new Google Cloud CI.
+The modules tested this week are given below:-
+    -[puppetlabs-vcsrepo](https://github.com/puppetlabs/puppetlabs-vcsrepo)
+    -[puppetlabs-exec](https://github.com/puppetlabs/puppetlabs-exec)
+    -[puppetlabs-facter_task](https://github.com/puppetlabs/puppetlabs-facter_task)
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_15.md:21]: Missing a space after the `-`
```suggestion
    - [puppetlabs-vcsrepo](https://github.com/puppetlabs/puppetlabs-vcsrepo)
    - [puppetlabs-exec](https://github.com/puppetlabs/puppetlabs-exec)
    - [puppetlabs-facter_task](https://github.com/puppetlabs/puppetlabs-facter_task)
```

File: _lifeofinternatpuppet/post_15.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Thank you to [Sheena](https://github.com/sheenaajay) for doing such an enjoyable session on module release prep and module release.
+The modules covered during the session were:-
+    -[puppetlabs-registry](https://github.com/puppetlabs/puppetlabs-registry) - (3.2.0)
+    -[puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) - (7.7.0)
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_15.md:26]: Missing a space after the `-`
```suggestion
    - [puppetlabs-registry](https://github.com/puppetlabs/puppetlabs-registry) - (3.2.0)
    - [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) - (7.7.0)
```

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]
Noticed some formatting errors. I would recommend testing this renders as you would expect in the browser by building and starting a local instance:
```ruby
bundle install
bundle exec jekyll build --future
bundle exec jekyll serve --future
```

File: _lifeofinternatpuppet/post_15.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Thank you to [Sheena](https://github.com/sheenaajay) for doing such an enjoyable session on module release prep and module release.
+The modules covered during the session were:-
+    - [puppetlabs-registry](https://github.com/puppetlabs/puppetlabs-registry) - (3.2.0)
+    - [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt) - (7.7.0)
+
+- Thanks to [Daiana](https://github.com/daianamezdrea) and [Paula](https://github.com/pmcmaw)for giving me the following PR's for review and merge
+    - PDK update for GA workflows[https://github.com/puppetlabs/puppetlabs-helm/pull/134]
+    - PDK update for GA workflow [https://github.com/puppetlabs/puppetlabs-kubernetes/pull/457]
+    - Delete i18n header(https://github.com/puppetlabs/iac/pull/153)
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_15.md:31]: These do not look like properly formatted links. What is the intention for these? Is it to look like this?

[PDK update for GA workflows](https://github.com/puppetlabs/puppetlabs-helm/pull/134)

Again, please note [my comment here](https://github.com/puppetlabs/iac/pull/155#pullrequestreview-550000540). Building and testing this locally would catch any formatting errors like this.

**Review by sanfrancrisko — CHANGES_REQUESTED** [review]

File: _lifeofinternatpuppet/post_15.md (hunk 4 above)
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_15.md:31]: @sanfrancrisko, I just added them for reference, but if you think that they are not necessary, then I will remove them. Personally, I don't think they were needed but just included them anyway :) Let me know either way of what to do!

Comment (Disha-maker) [reply _lifeofinternatpuppet/post_15.md:31]: I am just giving names of some of the PR's that I have reviewed and merged, but can include them as normal text and not hyperlinks with the https address removed too :)

Comment (sanfrancrisko) [reply _lifeofinternatpuppet/post_15.md:31]: You can include them if you want, but I'm not sure why you would enclose them in square braces, which made me think you intended these to be a hyperlink in MD format, e.g.
```ruby
[PDK update for GA workflow](https://github.com/puppetlabs/puppetlabs-kubernetes/pull/457)
```

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**